### DISCUSSION TITLE
bypy 最近經常發生上傳問題

### DISCUSSION THREAD

**Discussion by vientito** [houtianze/bypy#659] [Q&A]
Error accessing 'https://c.pcs.baidu.com/rest/2.0/pcs/file'
ForkPoolWorker-4: <E> [17:24:39] Function: _upload_slice_act
ForkPoolWorker-4: <E> [17:24:39] Website parameters: {'method': 'upload', 'type': 'tmpfile'}
ForkPoolWorker-4: <E> [17:24:39] Full URL: https://c.pcs.baidu.com/rest/2.0/pcs/file?method=upload&type=tmpfile&access_token=121.87105a55170697ed7444cbd98574e352.YlR7ks1V1QgoAveM6KNxR99JE7d-MRRqbLHw9Ue.zgjnpg
ForkPoolWorker-4: <E> [17:24:39] HTTP Response Status Code: 408
ForkPoolWorker-4: <E> [17:24:39] Error JSON returned:{'error_code': 31233, 'error_msg': 'read request body timeout'}
Error code: 31233
Error Description: read request body timeout
ForkPoolWorker-4: <E> [17:24:39] Website returned: {"error_code":31233, "error_msg":"read request body timeout"}

上傳有時非常停滯

ping 結果：

$ ping c.pcs.baidu.com
PING c.pcs.n.shifen.com (112.46.2.37) 56(84) bytes of data.
64 bytes from 112.46.2.37 (112.46.2.37): icmp_seq=4 ttl=44 time=55.2 ms
64 bytes from 112.46.2.37 (112.46.2.37): icmp_seq=9 ttl=44 time=55.1 ms
64 bytes from 112.46.2.37 (112.46.2.37): icmp_seq=14 ttl=44 time=55.1 ms
64 bytes from 112.46.2.37 (112.46.2.37): icmp_seq=17 ttl=44 time=55.1 ms
64 bytes from 112.46.2.37 (112.46.2.37): icmp_seq=18 ttl=44 time=55.1 ms
64 bytes from 112.46.2.37 (112.46.2.37): icmp_seq=19 ttl=44 time=55.4 ms
64 bytes from 112.46.2.37 (112.46.2.37): icmp_seq=20 ttl=44 time=55.1 ms
^C
--- c.pcs.n.shifen.com ping statistics ---
21 packets transmitted, 7 received, 66.6667% packet loss, time 20280ms
rtt min/avg/max/mdev = 55.055/55.150/55.394/0.115 ms

這會否 Baidu 侍服器問題？

**Comment by s0ren99** [comment]
你好，請問你的問題解決了嗎？
我遇到了同樣的問題，開了會員也上傳不動，也不能下載。ping了也有丟包...
我在用的biliup是白天上傳不了，凌晨可以，不知道bypy是不是也這樣，但今天凌晨忘記試了，而雙十一活動明天就沒了😭

**Comment by vientito** [comment]
问题大至与上傳時份有关(清早好似顺暢一些)，我也採用选项 -s 1M 使用少量一次性上傳，但下午時段开始差，我相信百度那边存在技术困难 ---- On Sat, 11 Nov 2023 17:30:29 +0900  ***@***.***  wrote ----
你好，請問你的問題解決了嗎？
我遇到了同樣的問題，開了會員也上傳不動，也不能下載。ping了也有丟包...
我在用的biliup是白天上傳不了，凌晨可以，不知道bypy是不是也這樣，但今天凌晨忘記試了，而雙十一活動明天就沒了😭
